demo/atom/imagePanel/playground

```diff
@@ -0,0 +1,69 @@
+const imageSrcLong = 'http://lorempicsum.com/simpsons/1000/100/1'
```

Comment:
* We should add examples of all use cases exposed in the zepplin.

Reply:
What example do you mean exactly? If it is the switched/replace example, it does not apply to this version as mentioned as comment in https://paper.dropbox.com/doc/SUI-Image-Panel-AjDu7OEMGBGaJaHiK8Vlf

I'll add a white text over black background example cause I don't see what else is missing

Reply:
The white text for instance, yes. Putting all the examples you make sure to see the behavior.
Every behavior/style that you develop should be documented by example.